solver/pb/caps.go

```diff
@@ -33,7 +33,7 @@ const (
 	CapSourceHTTP         apicaps.CapID = "source.http"
 	CapSourceHTTPChecksum apicaps.CapID = "source.http.checksum"
 	CapSourceHTTPPerm     apicaps.CapID = "source.http.perm"
-	CapSourceHTTPUIDGID   apicaps.CapID = "soruce.http.uidgid"
+	CapSourceHTTPUIDGID   apicaps.CapID = "source.http.uidgid"
```

Comment:
```suggestion
	CapSourceHTTPUIDGID           apicaps.CapID = "source.http.uidgid"
	CapSourceHTTPUIDGIDTypo   apicaps.CapID = "soruce.http.uidgid" // typo (v0.3 - v0.11), but unremovable for maintaining compatibility 
```

The typo cap must also remain enabled (see https://github.com/moby/buildkit/search?q=CapSourceHTTPUIDGID to find the code that has to be updated)